config/initializers/extends.rb

```diff
@@ -381,7 +381,9 @@ class Extends
     create_project_folder: 159,
     move_project_folder: 160,
     rename_project_folder: 161,
-    delete_project_folder: 162
+    delete_project_folder: 162,
+    change_user_role_on_experiment: 163,
```

Comment:
163 and 164 are already taken

Reply:
@okriuchykhin where are they used I can't find them. Is OK if I use `165` and `166`